example-apps/chatbot-rag-app/docker-compose.yml

```diff
@@ -1,16 +1,17 @@
 name: chatbot-rag-app
 
 services:
-  ingest-data:
+  create-index:
```

Comment:
renaed to match the flask command as having a different name here is confusing esp with `docker compose run` vs python